apps/alpha-test-app/src/views/HomeView.vue

```diff
@@ -111,13 +132,18 @@ const listboxOptions = [
         <p>Each onyx component should be used at least once in this page.</p>
 
         <div class="page__examples">
-          <OnyxButton v-if="show('OnyxButton')" label="Button" />
+          <OnyxButton
+            v-if="show('OnyxButton')"
+            label="Button"
+            :density="activeDensityOption.value"
+          />
 
           <template v-if="show('OnyxCheckboxGroup')">
             <OnyxCheckboxGroup
               v-model="checkboxState"
               headline="Checkbox Group"
               :options="dummyOptions"
+              :density="activeDensityOption.value"
```

Comment:
We don't need to set it on every component. We can just set the [density class](https://onyx.schwarz/development/density.html) on the parent div